CONTRIBUTING.md

```diff
@@ -41,7 +41,7 @@
 Contributing to Apache Commons CSV
 ======================
 
-You have found a bug or you have an idea for a cool new feature? Contributing code is a great way to give something back to
+You have found a bug, or you have an idea for a cool new feature? Contributing code is a great way to give something back to
```

Comment:
This is auto generated text so this is not the place to fix this. Can you revert please.